### ISSUE TITLE
[mat-list][a11y] Not possible to see focus indicator in high Contrast mode

### ISSUE THREAD

**Issue by stevenshchin** [angular/components#17366]
#### What is the expected behavior?

It should be possible to see the focus indicator on list items when Windows High Contrast mode is enabled.

#### What is the current behavior?

Only works in IE, doesn't work in FF and Chrome.

#### What are the steps to reproduce?

1. Turn on High Contrast mode
1. Navigate https://material.angular.io/components/list/overview
1. Hover on a list which provide hover/focus state. (For Angular Material 8.2.1, use https://material.angular.io/components/list/overview#selection-lists)

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?

Angular Material 8.2.1, Firefox & Chrome

IE
![mat-list-high-contrast-ie](https://user-images.githubusercontent.com/3030082/66636687-8e011b80-ebc6-11e9-9d33-69a3254fe639.png)

FF
![mat-list-high-contrast-ff](https://user-images.githubusercontent.com/3030082/66636675-88a3d100-ebc6-11e9-961f-f0d112ab9024.png)

**Comment by jelbourn** [comment]
@crisbeto I have something in mind for this

**Comment by crisbeto** [comment]
This is fixed in master.
